tests/resources/test_branch.py

```diff
@@ -94,7 +95,7 @@ def test_branch_register(session, collection, branch_path):
 def test_branch_get(session, collection, branch, branch_path):
```

Comment:
`branch` isn't used any more. If you ended up replacing all of the fixture uses with the Factory, you can probably 🪓  the fixture itself too. (I didn't check the whole file for uses of it though)

Reply:
There were a few other places I used it, but they weren't really necessary, so I ripped it out.